src/Home/Home.tsx

```diff
@@ -49,59 +75,280 @@ const Home = ({
     )
   }
 
-  const handleRequestPermission = () => {
+  const showBluetoothDisabledAlert = () => {
+    Alert.alert(
+      t("home.bluetooth.bluetooth_disabled_error_title"),
+      t("home.bluetooth.bluetooth_disabled_error_message"),
+      [
+        {
+          text: t("common.okay"),
+        },
+      ],
+    )
+  }
+
+  const { applicationName } = useApplicationInfo()
+  const appDownloadLink = "https://pathcheck.org"
```

Comment:
should this be coming in from a variable or constant?

Reply:
There's a card cut to route to the actual app download link, which will probably include getting this from the env.